### PR TITLE
Updated GitHub description and homepage URL to be https

### PR DESCRIPTION
**Pull request by bryancall**
Also added labels and being explicit on the options we already have setup

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -4,6 +4,24 @@ notifications:
   issues: issues@trafficserver.apache.org
   pullrequests: github@trafficserver.apache.org
 github:
+  description: "Apache Traffic Server™ is a fast, scalable and extensible HTTP/1.1 and HTTP/2.0 compliant caching proxy server."
+  homepage: https://trafficserver.apache.org/
+  lables:
```
Comment (shukitchan) [inline .asf.yaml:9]: labels ?

Comment (bryancall) [reply .asf.yaml:9]: Yes, it is part of the documentation from ASF.  I don't know how they are used.

https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features

Comment (shukitchan) [reply .asf.yaml:9]: i mean... isn't it a typos? you have "lables" instead.

File: .asf.yaml [hunk 2]
```diff
@@ -4,6 +4,24 @@ notifications:
   issues: issues@trafficserver.apache.org
   pullrequests: github@trafficserver.apache.org
 github:
+  description: "Apache Traffic Server™ is a fast, scalable and extensible HTTP/1.1 and HTTP/2.0 compliant caching proxy server."
```
Comment (masaori335) [inline .asf.yaml:7]: `HTTP/2`

**Pull request merged**